File: Itido-BMS/dart/bms_dart/lib/src/models/agreement.dart
class Agreement {
  int id;
  String name;

  Agreement({
    this.id,
    this.name,
  });

  factory Agreement.fromJson(json) {
    if (json == null) return null;
    return Agreement._fromJson(json);
  }

  Agreement._fromJson(json)
      : this.id = json['id'],
        this.name = json['name'];

  Map<String, dynamic> toMap() => {
        'id': this.id,
        'name': this.name,
      };
}

File: Itido-BMS/RentAPI/Rent/Models/TimePlanning/Agreement.cs
using System.Collections.Generic;
using System.Linq.Expressions;
using System;


namespace Rent.Models.TimePlanning
{

    public class Agreement
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public virtual ICollection<Contract> Contracts { get; set; }


        public static Expression<Func<Agreement, dynamic>> BasicDTO()
        {
            return v => v != null ?
            new
            {
                v.ID,
                v.Name

            } : null;
        }

    }

}
